packages/react/src/themes/teams/components/Button/buttonVariables.ts

```diff
@@ -74,7 +75,7 @@ export default (siteVars: any): ButtonVariables => {
     maxWidth: pxToRem(280),
     borderRadius: pxToRem(2),
     circularRadius: pxToRem(999),
-    fontWeight: siteVars.fontWeightSemibold,
+    buttonFontWeight: siteVars.fontWeightSemibold,
```

Comment:
I know there was already discussion about the name, but maybe `contentFontWeight` will be better name, as it is not applied to the icon part. In my opinion` buttonFontWeight` is not any different then `fontWeight`, as it is used inside the button, but doesn't explain that it is applied only in the content.